.github/workflows/cherry-pick-pr.yml

```
@@ -0,0 +1,24 @@
+name: Cherry-pick PRs from next to master
+
+on:
+  pull_request:
+    branches:
+      - next
+    types: ['closed']
+
+jobs:
+  cherry_pick_release_v1_0:
+    runs-on: ubuntu-latest
+    name: Cherry-pick into master
+    if: contains(github.event.pull_request.labels.*.name, 'needs cherry-pick')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cherry-pick into master
+        uses: carloscastrojumo/github-cherry-pick-action@v1.0.1
```

We need to pin the versions of the actions: https://github.com/mui/mui-x/security/code-scanning/17

```suggestion
        uses: carloscastrojumo/github-cherry-pick-action@6523ab0b6ebe06801bd3933817017136961abd3c # v1.0.4
```